src/ophys_etl/modules/suite2p_registration/schemas.py

```diff
@@ -94,6 +94,29 @@ class Suite2PRegistrationInputSchema(argschema.ArgSchema):
         default=8,
         description="Maximum number of iterations to preform when creating a "
                     "reference image.")
+    remove_empty_frames = argschema.fields.Boolean(
+        required=False,
+        default=False,
+        allow_none=False,
+        description="Some movies arrive with otherwise quality data but have "
+                    "but contain a set of frames that are empty and contain "
+                    "pure noise. When processed, these frames tend to receive "
+                    "large random shifts that throw off motion border "
+                    "calculation. Turning on this setting on removes a "
```

Comment:
"Turning on this setting on"

Reply:
Also: can you mention that the code auto detects which frames should be removed. "removes a defined set of frames from the movie" makes it sound to me like the user has to specify the frames to remove.

Reply:
My plan initially changed as I developed this. Tying into the comment below, the idea here is that if you know you have frames you want to remove, you can just specify them or you can have the code try to find them for you. The naming and docstrings likely confuse this a bit. I'll give it a shot at changing the naming and modifying the docs but welcome suggestions.

Reply:
'auto_remove_empty_frames'?